gladier_xpcs/tools/pre_publish.py

```diff
@@ -25,6 +25,16 @@ def pre_publish_gather_metadata(**data):
     from pilot.client import PilotClient
     from pilot.exc import PilotClientException, FileOrFolderDoesNotExist
 
+    try:
+        import boost_corr
+        boost = True
+        boost_version = boost_corr.__version__
+        gpu = data.get('gpu_flag','0')
+    except:
+        boost = False
+        boost_version = 0
+        gpu = None
+        
```

Comment:
We should add this in the XPCS gather function instead. Pre-publish would be a good one to remove in favor of the bulit-in gladier tool.